packages/node_modules/@ciscospark/widget-recents/src/enhancers/listeners.js

```diff
@@ -69,6 +69,11 @@ function processActivity(activity, space, props) {
     spacesList
   } = props;
 
+  // Reply activities are not currently supported
+  if (activity.type === API_ACTIVITY_TYPE.REPLY) {
```

Comment:
is this quoting?

Reply:
No, this is threaded replies. We probably need a feature epic for click quoting.

Reply:
quoting kinda works actually. they are doing a mark ups with `>`.